qa/workunits/ceph-helpers.sh

```diff
@@ -123,6 +132,16 @@ function teardown() {
     rm -fr $dir
 }
 
+function teardown_error() {
```

Comment:
@tchaikov 
This is in preparation that we start doing some sensible things when a test actually fails. We then do need the poor remains of the carcass to see what happened. But that can go into another commit/pull as well.